src/irmin/object_graph.mli

```diff
@@ -40,10 +40,20 @@ module type S = sig
     min:vertex list ->
     max:vertex list ->
     unit ->
-    t Lwt.t
-  (** [closure min max pred] creates the clansitive closure of [max]
+    (t, [ `Disconnected of vertex | `Empty_max ]) result Lwt.t
+  (** [closure min max pred] creates the transitive closure of [max]
       using the precedence relation [pred]. The closure will not
-      contain any keys before the the one specified in [min]. *)
+      contain any keys before the the one specified in [min].
```

Comment:
Hmm, this seems false to me. Am I right in thinking that the closure _will_ include ancestors of minimal nodes provided that there are paths to those nodes that don't go via the minimal nodes.

Reply:
Good catch, rephrased it.